plugins/catalog-backend/openapi.yaml

```diff
@@ -413,8 +533,17 @@ paths:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/RefreshOptions'
```

Comment:
Request bodies shouldn't generally not be refs unless they are reused in multiple places. If they're reused they should be keyed to a `requestBodies` key under `components` not just in the `components.schema` key. 

https://swagger.io/docs/specification/describing-request-body/